frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteGraphTable.java

```diff
@@ -162,14 +164,22 @@ public void refreshModel(Path macheteFilePath, boolean isMacheteFilePresent) {
     if (!isMacheteFilePresent) {
       setTextForEmptyGraph(
           "There is no machete file (${macheteFilePath}) for this repository.",
-          "Please use 'git machete discover' CLI command to automatically create machete file.");
+          "Create & open machete file", getOpenMacheteFileActionAsRunnable());
       LOG.info("Machete file (${macheteFilePath}) is absent");
     }
 
     repaint();
     revalidate();
   }
 
+  @UIEffect
+  private Runnable getOpenMacheteFileActionAsRunnable() {
```

Comment:
Hmm I think making this method `void` and using `() -> getOpenMacheteFileActionAsRunnable()` in line 167 would be more natural